fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/ButtonTokens.kt

```diff
@@ -30,9 +30,17 @@ enum class ButtonSize {
     Large
 }
 
```

Comment:
has this been specified in the specs?
Else we can just accept values instead of creating enum

Reply:
It hasn't been specified; I chose to implement it this way to make it similar to the other control tokens

Reply:
I think we can keep it open ended to allow greater level of customization, else user will be restricted with these 3-4 values of enum only

Reply:
Updated